modules/social_features/social_user_export/src/Plugin/Action/ExportUser.php

```diff
@@ -59,11 +59,11 @@ class ExportUser extends ActionBase implements ContainerFactoryPluginInterface {
    *   The plugin implementation definition.
    * @param \Drupal\user\PrivateTempStoreFactory $temp_store_factory
    *   The tempstore factory.
-   * @param \Drupal\Core\Session\AccountInterface $current_user
-   *   Current user.
+   * @param \Drupal\Core\Session\AccountInterface $currentUser
```

Comment:
Please keep the snake_case here which is usual for function variable names. camelCase is usually reserved for class properties.